test/plugin/scenarios/elasticjob-3.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: elasticjob-3.x-scenario
+  segmentSize: ge 2
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /elasticjob-3.x-scenario/case/ping
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 12
+      isError: false
+      spanType: Exit
+      peer: localhost:8080
+      skipAnalysis: false
+      tags:
+        - {key: http.method, value: GET}
+        - {key: url, value: 'http://localhost:8080/elasticjob-3.x-scenario/case/ping'}
+    - operationName: ElasticJob/simpleJob
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Unknown
+      startTime: not null
+      endTime: not null
+      componentId: 24
+      isError: false
+      spanType: Entry
+      peer: ''
+      skipAnalysis: false
+      tags:
+        - {key: item, value: '0'}
+        - {key: shardingTotalCount, value: '1'}
+        - {key: taskId, value: not null}
+        - {key: shardingItemParameters, value: '{0=Beijing}'}
```

Comment:
@tristan-tsl Could you check whether these are recommended and important messages for the elastic job users? Any thing more?